### ISSUE TITLE
Last contact item cannot be fully viewed using keyboard navigation alone

### ISSUE THREAD

**Issue by mongj** [mongj/pe#6]
When the keyboard is used for navigating the contact list, the the list will stop scrolling when the focus is on the last card and if scrolling further means cutting the top card.

![Screenshot 2024-11-15 at 5.02.46 PM.png](https://raw.githubusercontent.com/mongj/pe/main/files/e6ad11e6-fa66-41fd-ad00-da962a46e11c.png)
(clicking `down` arrow key when the last card is selected here doesn't do anything)

The expected behavior is that whichever card is selected should always be fully visible.

This is a feature flaw as the up/down key navigation does not work as intended for users who prefer CLI.

I put this as low severity since users can still use the info command to see the full details once the correct contact is selected, or use the mouse to scroll all the way down

**Comment by nus-pe-script** [comment]
# Team's Response
Hello! Thank you for your bug. We have declared this bug as out of scope as you can view the full details via the information window and there are also ways to access the person card such as clicking on the personcard, as well as scrolling down. You can also resize the window to full screen to view the details. As such, we have declared this as out of scope. Thank you!

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/5445ad3b-df49-44d3-b14c-ddf41d93c40a.png)
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
